src/app/shared/components/wrapper/wrapper.component.spec.ts

```diff
@@ -1,14 +1,20 @@
 import { ComponentFixture, TestBed } from '@angular/core/testing';
 
 import { WrapperComponent } from './wrapper.component';
+import { RouterTestingModule } from '@angular/router/testing';
+import { CUSTOM_ELEMENTS_SCHEMA } from '@angular/core';
 
 describe('WrapperComponent', () => {
   let component: WrapperComponent;
   let fixture: ComponentFixture<WrapperComponent>;
 
   beforeEach(async () => {
     await TestBed.configureTestingModule({
-      declarations: [ WrapperComponent ]
+      imports: [RouterTestingModule],
+      declarations: [
+        WrapperComponent,
+      ],
+      schemas: [CUSTOM_ELEMENTS_SCHEMA]
     })
     .compileComponents();
 
```

Comment:
Add here also 1 test, that this component render header and footer also add 1 test for Breadcrumbs (that Courses text was rendered)

Reply:
fixed